wpeverest/user-registration
# UR-416 Introduced  - Quick Setup Tab

### All Submissions:






### Changes proposed in this Pull Request:
Previously we didn't have a method of notifying users whether or not they had completed all the steps of setting up our plugin. They will run setup wizard, some may also skip them and later they didn't have method to rerun setup wizard and also if setup wizard completed still spam protection measures which are important is not included in setup wizard and user would not set it up and complain about getting spams in deactivation reasons.

Considering these cases, a new Quick Setup Tab has been introduced in the User Registration -> Settings Right Sidebar, which lists the most important steps needed to properly setup our plugin where the option to continue the setup wizard is given if skipper previously. Until the user completes all the steps, the setup tab will be displayed, once completed it will be removed.

### How to test the changes in this Pull Request:

1.  Setup user registration in the new site.
2. Skip the setup wizard at some point.
3. Go to Settings, a Quick setup tab will be displayed, and an option to continue the setup wizard will be shown. Click on it and verify if it continues from the point it is skipped.
4. Also other options in the list will be present, complete each requirement, and see of the item in the list is shown as completed.
5. If all steps are completed then the Quick Setup Tab should be hidden.

### Types of changes:




### Other information:




### Changelog entry

>  Introduced  - Quick Setup Tab.
